src/NetworkTestWidget/useTestRunner.ts

```diff
@@ -30,7 +30,8 @@ export default function useTestRunner() {
 
         try {
           const voiceToken = await getVoiceToken();
-          testResults.results.preflight = await preflightTestRunner(region, voiceToken);
+          const iceServers = await getTURNCredentials();
```

Comment:
Is this regionalized?

Reply:
I didnt check actually. It uses the same method as the bandwidth test. @timmydoza, does it know which region?

Reply:
@aymenn I updated the PR to regionalize the iceServers. I also merged in the `main` branch and updated tests.